shared/js/project.js

```diff
@@ -127,7 +111,7 @@ module.exports = {
       $('.project_updated_at').text("Not saved"); // TODO i18n
     }
   },
-  appToProjectUrl: function () {
+  appToProjectUrl_: function () {
```

Comment:
Ah never mind, it looks like this method actually gets used in https://github.com/code-dot-org/code-dot-org/blob/f46ba82950c6eda2741b920c9fe5288c69b42f9f/dashboard/app/assets/javascripts/header.js.erb#L166.  I don't think we want the trailing underscore on this one.